psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -79,15 +79,16 @@ class EmpiricalDispersion(object):
         corresponds to a defined, named, untweaked "functional-dashlevel"
         set with a citation. Otherwise, empty string.
     dashcoeff_supplement : dict
-        See description in `qcengine.programs.empirical_dispersion_resources.from_arrays`. Used
+        See description in `qcengine.programs.dftd3.dashparam.from_arrays`. Used
         here to "bless" the dispersion definitions attached to
         the procedures/dft/*_functionals-defined dictionaries
         as legit, non-custom, and of equal validity to
-        `qcengine.programs.empirical_dispersion_resources.dashcoeff` itself for purposes of
+        `qcengine.programs.dftd3.dashparam.dashcoeff` itself for purposes of
         validating `fctldash`.
-    engine : {'libdisp', 'dftd3', 'nl', 'mp2d'}
+    engine : {'libdisp', 'dftd3', 'nl'}
```

Comment:
mp2d is legit. https://github.com/psi4/psi4/blob/master/psi4/driver/procrouting/empirical_dispersion.py#L88